Converters/Autocad/Speckle.Converters.AutocadShared/ToSpeckle/Geometry/HatchToSpeckleConverter.cs

```diff
@@ -78,25 +78,19 @@ public SOG.Region Convert(ADB.Hatch target)
 
   private ADB.Curve PolylineFromLoop(ADB.HatchLoop loop)
```

Comment:
Also this is a misleading method: should probably call it `ConvertHatchLoopToCurveEntity`. The reason for needing to convert hatch loops to curve entities should be explained as in the above comments